docs/using-scylla/cassandra-compatibility.rst

```diff
@@ -21,25 +21,25 @@ For all tables below, the following convention is used:
 Interfaces
 ----------
 
-==============================  =============================================  =============
-Apache Cassandra                Version                                        Comments
-==============================  =============================================  =============
-CQL                             CQL spec 3.4.0, Native protocol v4 (C* 3.11)   More below
-------------------------------  ---------------------------------------------  -------------
+==============================  =====================================================  =============
+Apache Cassandra                Version                                                Comments
+==============================  =====================================================  =============
+CQL                             CQL spec |cql-version|, Native protocol v4 (C* 3.11)   More below
+------------------------------  -----------------------------------------------------  -------------
 Thrift                          Compatible with Cassandra 2.1
-------------------------------  ---------------------------------------------  -------------
-SSTable format (all versions)   3.11(mc / md ), 2.2(la), 2.1.8 (ka)            md format is supported in Scylla 4.3, 2021.1.0 and later
-------------------------------  ---------------------------------------------  -------------
-JMX                             3.11                                           More Below
-------------------------------  ---------------------------------------------  -------------
+------------------------------  -----------------------------------------------------  -------------
+SSTable format (all versions)   3.11(mc / md ), 2.2(la), 2.1.8 (ka)                    md format is supported in Scylla 4.3, 2021.1.0 and later
```

Comment:
We will also support the `me` format form version 5.1 / 2022.1.
We should make the change in master and backport it to the 5.1 branch.

Reply:
@bhalevy I'm going to update the documentation according to the discussion below, but I'll skip this information as there is a separate issue opened to do that: https://github.com/scylladb/scylla-docs/issues/3990
It's going to be documented for the upcoming OSS and Enterprise releases.